server/util.go

```diff
@@ -412,3 +416,46 @@ func openTraceWriter(traceWriterFile string) (w io.Writer, err error) {
 		0o666,
 	)
 }
+
+// DefaultBaseappOptions returns the default baseapp options provided by the Cosmos SDK
+func DefaultBaseappOptions(appOpts types.AppOptions) []func(*baseapp.BaseApp) {
```

Comment:
... would have been nice if they had wrapped this as a single `func(*baseapp.Baseapp)` return value so that you don't have to `...` after it, but oh well.